.github/workflows/codeql-query.yml

```diff
@@ -165,3 +165,93 @@ jobs:
 
       - name: Combine results
         uses: ./combine-results
+
+  update-repo-tasks-statuses-cancelled:
+    runs-on: ubuntu-latest
+    if: cancelled() && github.event.inputs.variant_analysis_id != ''
+    needs:
+      - run
+    permissions:
+      security-events: write # This permission can be removed soon
+      contents: write
+
+    steps:
+      # To use the action in this repository, we need to check out the repository
+      - name: Checkout codeql-variant-analysis-action repository
+        uses: actions/checkout@v3
+        with:
+          repository: "github/codeql-variant-analysis-action"
+          ref: ${{ github.event.inputs.action_repo_ref }}
+
+      - name: Mask download URL
```

Comment:
We should mask the signed URL in these jobs too. See https://github.com/github/codeql-variant-analysis-action/blob/84ca3c82f1a66d4f8409ff3ff95881299c401ee5/.github/workflows/codeql-query.yml#L49